mail-filter/postsrsd/files/postsrsd.init-r2

```diff
@@ -1,10 +1,10 @@
 #!/sbin/openrc-run
-# Copyright 2015 Gentoo Foundation
```

Comment:
By git mving the old init file, the old ebuild revision won't work. Need to copy it.

Reply:
please take another look. postsrsd.init was an orphan file.

postsrsd-1.11.ebuild references postsrsd.init-r1
the new postsrsd-1.11-r1.ebuild references postsrsd.init-r2 which was copied from postsrsd.init-r1

Reply:
Got it :)